website/source/guides/operations/deployment-guide.html.md

```diff
@@ -187,17 +180,14 @@ server {
 ```
 
 - [`server`](/docs/configuration/server.html#enabled) - Specifies if this agent should run in server mode. All other server options depend on this value being set.
-- [`bootstrap-expect`](/docs/configuration/server.html#bootstrap_expect) - This flag provides the number of expected servers in the datacenter. Either this value should not be provided or the value must agree with other servers in the cluster.
+- [`bootstrap_expect`](/docs/configuration/server.html#bootstrap_expect) - The number of expected servers in the datacenter. Either this value should not be provided or the value must agree with other servers in the cluster.
```

Comment:
Datacenter here is ambiguous and is used colloquially meaning `cluster` here.  I'd change it to cluster, or use "nomad region", the proper technical nomad jargon.

```suggestion
- [`bootstrap_expect`](/docs/configuration/server.html#bootstrap_expect) - The number of expected servers in the cluster. Either this value should not be provided or the value must agree with other servers in the cluster.
```